lib/datastore/transaction.js

```diff
@@ -143,10 +141,10 @@ Transaction.prototype.rollback = function(callback) {
  * Commit the remote transaction and finalize the current transaction instance.
  *
  * @param {function} callback - The callback function.
+ * @private
```

Comment:
Privatize `transaction#commit` without changing the name yet. Should I go ahead and change it to `commit_` to follow convention?